### ISSUE TITLE
Make delimiters automatically create a tag

### ISSUE THREAD

**Issue by Enigmaticall** [yairEO/tagify#1187]
Delimiters today work, but only once you've started typing the next word. If I have a space as a delimiter, I would like it to instantly create a tag so the user gets the feedback that it worked before they start typing the next tag. I have received this feedback from multiple users, would that be possible?

I am trying to switch tags immediately after any of the following characters are typed:
delimiters: ',|; '

Thanks in advance!

**Comment by yairEO** [comment]
Duplicate of #1049

**Comment by EtudiantFabre** [comment]
On ne peut pas préciser deux délimiteurs au lieu d'un seul ?

**Comment by yairEO** [comment]
@EtudiantFabre - why did you not write in English? anyway:

<img width="846" height="100" alt="Image" src="https://github.com/user-attachments/assets/64fa9784-0975-41a2-bcc6-7de6ff42216a" />
